feat!(exporter/blackbox): use required name attr instead label for target block (#5945)
grafana/agent: CHANGELOG.md

Main (unreleased)
-----------------

### Breaking changes

> - `target` block in `prometheus.exporter.blackbox` requires mandatory `name`
>   attribute instead of label block previously. (@hainenber)

Reviewer: ```suggestion
- The `target` block in `prometheus.exporter.blackbox` requires a mandatory `name`
  argument instead of a block label. (@hainenber)
```
